### PR TITLE
Fixing the Tier4b issues

### PR DESCRIPTION
**Pull request by Gangadhar-ibm-lab**
Fixing the Tier 4b issues

### CODE REVIEW

**Review by vavuthu — APPROVED** [review]

**Review by petr-balogh — DISMISSED** [review]

**Comment by petr-balogh** [comment]
We need approve from someone from brown squad

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/utility/ibmcloud.py [hunk 1]
```diff
@@ -357,7 +358,19 @@ def get_node_by_attached_volume(self, volume):
...
+            worker_nodes = get_nodes(node_type="worker")
+            for worker_node in worker_nodes:
+                logger.info(
+                    f"worker node id is:{worker_node.get()['metadata']['labels']['ibm-cloud.kubernetes.io/worker-id']}"
+                )
+                if (
+                    worker_node.get()["metadata"]["labels"][
+                        "ibm-cloud.kubernetes.io/worker-id"
+                    ]
+                    == worker_id
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:370]: maybe this line can be merged together with the line above? Like:
```
] == worker_id
```

File: ocs_ci/utility/ibmcloud.py [hunk 2]
```diff
@@ -357,7 +358,19 @@ def get_node_by_attached_volume(self, volume):
...
+                ):
+                    logger.info(f"return worker node is:{worker_node}")
```
Comment (vavuthu) [inline ocs_ci/utility/ibmcloud.py:372]: log worker node name instead of instance

File: ocs_ci/utility/ibmcloud.py (hunk 1 above)
Comment (Gangadhar-ibm-lab) [reply ocs_ci/utility/ibmcloud.py:370]: black is formatting like this and tried several ways.

File: ocs_ci/utility/ibmcloud.py (hunk 2 above)
Comment (Gangadhar-ibm-lab) [reply ocs_ci/utility/ibmcloud.py]: Done the changes.

File: ocs_ci/utility/ibmcloud.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/utility/ibmcloud.py:370]: ok, no problem, if black likes this way than it's fine.. Just to my eyes I would prefer to do not start line with operator == . :)

**Review by petr-balogh — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Pull request merged**